modules/dists/CyclicDist.chpl

```diff
@@ -54,19 +54,30 @@ config param disableCyclicLazyRAD = defaultDisableLazyRADOpt;
 //   disableCyclicLazyRAD
 //
 /*
-This Cyclic distribution maps indices to locales in a round-robin pattern
-starting at a given index.
 
-Formally, consider a Cyclic distribution with:
+The ``cyclicDist`` distribution distributes d-dimensional indices to
```

Comment:
This might be heavy on alliteration. Consider: "The `cyclicDist` distribution uses a round-robin partitioning to distribute d-dimensional indices across a d-dimensional array of locales, starting from a given index".